src/main/resources/assets/clientcommands/lang/pl_pl.json

```diff
@@ -0,0 +1,124 @@
+  "commands.cenchant.success": "Rozpoczynam manipulowanie zaklęciami",
+  "commands.cenchant.uncracked": "Ziarno musi być złamane",
+
+  "commands.cfind.found.left": "Znaleziono %1$s na ",
+  "commands.cfind.found.right": ", %2$.2f bloków dalej",
+  "commands.cfind.noMatch": "Żadne entity nie pasuje twoim wymaganiom",
+  "commands.cfind.success": "%d entity pasuje twoim wymaganiom",
+
+  "commands.cfindblock.notFound": "Nie znaleziono takiego bloku",
+  "commands.cfindblock.success.left": "Najbliższe trafienie jest na ",
+  "commands.cfindblock.success.right": ", %1$.2f bloków dalej",
+
+  "commands.cfinditem.match.left": "%dx %s znaleziono na ",
+  "commands.cfinditem.match.right": "",
+  "commands.cfinditem.starting": "Szukanie %s",
+  "commands.cfinditem.starting.keepSearching": "Ciągłe szukanie %s.",
+  "commands.cfinditem.total": "Znaleziono %dx %s",
+
+  "commands.crender.entities.success": "Reguła renderowania entity została zaktualizowana",
+
+  "commands.cscript.notFound": "Skrypt \"%s\" nie istnieje",
+  "commands.cscript.reload.success": "Przeładowanie Skryptów",
+  "commands.cscript.run.success": "Wykonano skrypt pomyślnie",
+
+  "commands.ctask.list.noTasks": "Nie ma w tej chwili działających zadań.",
+  "commands.ctask.list.success": "Działające zadania: %d",
+  "commands.ctask.stop.noMatch": "Żadnych pasujących zadań",
+  "commands.ctask.stop.success": "Zatrzymano %d zadań",
+
+  "commands.ctemprule.list.header": "Listowanie %d Tymczasowych reguł:",
+  "commands.ctemprule.reset.success": "Tymczasowa reguła %s została zresetowana do wartości %s",
+  "commands.ctemprule.set.success": "Tymczasowa reguła %s została zaktualizowana do wartości %s",
+
+  "commands.cwiki.failed": "Nie można było otrzymać zawartości z wiki",
+
+  "commands.client.blockpos": "(%d, %d, %d)",
+  "commands.client.cancel": "Anuluj",
+  "commands.client.crack": "Złam",
+  "commands.client.expectedRegex": "Nieprawidłowe wyrażenie regularne %s",
+  "commands.client.notClient": "To nie jest polecenie klienta.",
+
+  "enchCrack.state": "Stan ziarna zaklęć: %s",
+  "enchCrack.state.uncracked": "Nie złamano",
+  "enchCrack.state.cracking": "W trakcie",
+  "enchCrack.state.cracked": "Złamano",
+
+  "enchCrack.enchantments": "Zaklęcia:",
+  "enchCrack.clues": "Podpowiedzi:",
+  "enchCrack.slot": "Slot %d:",
+
+  "enchCrack.xpSeed.one": "Ziarno XP: %08X",
+  "enchCrack.xpSeed.many": "Liczba możliwych ziarn XP: %d",
+  "enchCrack.playerRNGSeed.many": "Liczba możliwych RNG gracza: %d",
+  "enchCrack.addInfo": "Dodaj Informacje",
+
+  "enchCrack.insn.dummy": "Zaklnij zbędny przedmiot",
+  "enchCrack.insn.ready": "Twoje ziarno zaklęcia jest gotowe",
+  "enchCrack.insn.bookshelves": "Półki na książki: %d",
+  "enchCrack.insn.slot": "W Slocie: %d",
+  "enchCrack.insn.itemThrows": "Ile musisz wyrzucić przedmiotów: %d (about %f seconds)",
+  "enchCrack.insn.itemThrows.noDummy": "Nie potrzebujesz zbędnego przedmiotu",
```

Comment:
Google translate tells me that "zbędnego" means "unnecessary". The English is more trying to hint at a "throwaway" item more than an unnecessary one. Putting "throwaway" into Google translate and looking at the adjective, it's giving me "do jednorazowego użytku". Would that be more suitable?

Reply:
Polish language is very difficult to translate and as i said we have many meanings. "do jednorazowego użytku" means "one time use" and "zbędnego" means that you're not gonna use that enchanted item and wanna disenchant that in order to use it again as a dummy. Im not saying that "do jednorazowego użytku" does not fit at all. But i think that "zbędnego" fits perfectly. Short Answer: Both are correct.

Reply:
Zbędny also means something with no value as wooden_sword

Reply:
I'll take your word for it, just checking